cookbook/src/pages/prerequisites.mdx

```diff
@@ -7,17 +7,18 @@ description: Prerequisites and supported software
 
 Node.js is a prerequisite for installing the Merative Social Program Managment UI Addon Development Environment and for developing and deploying your application pages.
```

Comment:
Another one for Elaine I think but we still have reference to Social Program Management across docs.. Probably a separate PR ant not specific to this one but probably need to change references to Curam from  'Social Program Management' or 'SPM'